### PR TITLE
Delete SEE ALSO section under content/zh/docs/reference/setup-tools/k…

### PR DESCRIPTION
**Pull request by ttonline6**
Delete SEE ALSO section under content/zh/docs/reference/setup-tools/kubeadm/generated/.

### CODE REVIEW

**Comment by ttonline6** [comment]
"Delete SEE ALSO section under content/zh/docs/reference/setup-tools/kubeadm/generated/.

**Comment by tanjunchen** [comment]
ref:https://github.com/kubernetes/website/issues/18441

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_etcd.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-<!-- 
+<!--
 ### Synopsis
 -->
 ### 概要
```
Comment (xichengliudui) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_etcd.md]: 与上方注释调换位置

**Pull request merged**